pkg/secrets-store/nodeserver_test.go

```diff
@@ -52,6 +51,9 @@ func getTestTargetPath(pattern string, t *testing.T) string {
 	if err != nil {
```

Comment:
Could we pass pattern to `tmpdir.New()` and use that here? We should use the tmpdir package for all the tests that create tmpdir.

Reply:
Perfect! This makes it easier to update the logic in a single function.